source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,298 @@
+value of a matching document from ``Bagels N Buns`` to ``2 Bagels 2 Buns``:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateOneSubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateOne(eq("name", "Bagels N Buns"), set("name", "2
+    Bagels 2 Buns")).subscribe(updateOneSubscriber);
+    updateOneSubscriber.await();
+
+Update Multiple Documents
+-------------------------
+
+To update multiple documents from a MongoDB collection, call the ``updateMany()``
+method and pass your query filter. Then, pass your
+update operators as the second argument to the ``updateMany()`` method. After you call the
+``updateMany()`` method, chain the ``subscribe()`` method and pass a subscriber
+as an argument.
+
+The following example updates all documents with a ``cuisine`` value of
+``Pizza`` to have a ``cuisine`` value of ``Pasta``:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateMany(eq("cuisine", "Pizza"), set("cuisine", "Pasta")).subscribe(updateManySubscriber);
+    updateManySubscriber.await();
+
+Customize the Update Operation
+------------------------------
+
+The ``UpdateOptions`` class contains methods that modify
+the behavior of update methods. To use the ``UpdateOptions``
+class, construct a new instance of the class, then call one or more of its methods
+to modify the update operation. You can chain these method calls together.
+To modify the behavior of the update operation, pass the class instance and
+chained method calls as the second argument to the
+``updateOne()`` or ``updateMany()`` method.
+
+You can use the following optional methods in the ``updateOptions`` class
+to modify an update method:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``arrayFilters(List<? extends Bson> arrayFilters)``
+     - | A list of filters that specifies which array elements an update applies
+         to.
+
+   * - ``bypassDocumentValidation(Boolean bypassDocumentValidation)``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+      
+   * - ``collation(Collation collation)``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the MongoDB Server manual.  
+
+   * - ``comment(Bson comment)``
+     - | A ``Bson`` comment to attach to the operation. For more information, see the :manual:`insert command
```

Comment:
S: similar comment as above - I think starting with a verb makes this more descriptive of the method itself
```suggestion
     - | Attaches a ``Bson`` comment to the operation. For more information, see the :manual:`insert command
```

Reply:
Applies to the other table descriptions (comment(String comment), let(Bson variables))